cmd/snap-update-ns/utils.go

```diff
@@ -665,7 +665,7 @@ func applyProfile(snapName string, currentBefore, desired *osutil.MountProfile,
 	var changesMade []*Change
 	for _, change := range changesNeeded {
 		logger.Debugf("\t * %s", change)
-		synthesised, err := changePerform(change, as)
+		synthesised, err := change.Perform(as)
```

Comment:
there are a couple more changePerform in this file that probably should be addressed too for consistency